Source/WebCore/page/NavigatorLoginStatus.h

```diff
@@ -26,31 +26,34 @@
 #pragma once
 
 #include "Supplementable.h"
+#include <wtf/WeakPtr.h>
```

Comment:
This should be removed.

Reply:
I believe we need this as we are extending  public CanMakeWeakPtr<NavigatorLoginStatus>. Correct me if I'm wrong

Reply:
Oh you're right.